test/utilities/new_versions.jl

```diff
@@ -135,11 +135,7 @@ end
         apply(gl_pr_patch) do
             result, n = CompatHelper.create_new_pull_request(
                 GitLab.GitLabAPI(),
-                GitLab.Project(;
-                    owner=GitLab.User(; name="username"),
-                    name="repo",
-                    path_with_namespace="owner/repo",
-                ),
+                GitLab.Project(; owner=GitLab.User(; name="username"), name="repo"),
```

Comment:
I think this should be changed back to whats in v3-dev, or more so you just need to add the `path_with_namespace` argument